docs/tutorial/views.rst

```diff
@@ -80,14 +92,29 @@ rule:
        natToBin (j + j)     | Even = False :: natToBin j
        natToBin (S (j + j)) | Odd  = True  :: natToBin j
 
-The value of the result of ``parity k`` affects the form of ``k``,
-because the result of ``parity k`` depends on ``k``. So, as well as
-the patterns for the result of the intermediate computation (``Even``
-and ``odd``) right of the ``|``, we also write how the results
-affect the other patterns left of the ``|``. Note that there is
-a function in the patterns (``+``) and repeated occurrences of
-``j``—this is allowed because another argument has determined the form
-of these patterns.
+The value of ``parity k`` affects the form of ``k``, because the
+result of ``parity k`` depends on ``k``. So, as well as the patterns
+for the result of the intermediate computation (``Even`` and ``odd``)
```

Comment:
Odd should be capitalized? :)

Reply:
Thanks, fixed in f0b1d501c3db2668c0fe1625f7821cae51cb2485 .